mcsema/BC/External.h

```diff
@@ -19,7 +19,10 @@
 
 namespace llvm {
 namespace CallingConv {
+#if !(LLVM_VERSION_MAJOR == 3 && LLVM_VERSION_MINOR == 5)
```

Comment:
You know what; it'd easier to just `#include` LLVM's calling convention header here, as it doesn't actually `#include` much anyway.